.github/workflows/sync-bioconductor.yml

```
@@ -0,0 +1,54 @@
+
+    - name: Configure git
+      run: |
+        git config user.name "github-actions[bot]"
+        git config user.email "github-actions[bot]@users.noreply.github.com"
+        
+    - name: Sync with Bioconductor
+      run: |
+        git remote remove upstream || true
+        git remote add upstream git@git.bioconductor.org:packages/MSstatsBioNet.git
+        git fetch upstream
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)